src/features/TenantSupport/TenantSupport.tsx

```diff
@@ -39,13 +41,17 @@ const issues = [
   },
   {
     category: 'Maintenance Issue',
+    backendCategory: 'maintenance',
+
```

Comment:
Question: wonder how this extra line ended up here? Should have been caught w linting